ras_party/controllers/queries.py

```diff
@@ -169,3 +170,17 @@ def query_enrolment_by_survey_business_respondent(respondent_id, business_id, su
                                                     Enrolment.business_id == business_id,
                                                     Enrolment.survey_id == survey_id)).first()
     return response
+
+
+def query_enrolment_by_survey_business(business_id, survey_id, session):
+    """
+    Query to return enrolments based on business id and survey
```

Comment:
I think the method name should reflect it's a count, maybe `count_enrolments_by_survey_business`. Also, the docstring needs updating to reflect it's a count.